sopel/modules/admin.py

```diff
@@ -182,7 +182,10 @@ def set_config(bot, trigger):
     If value is None, the option will be deleted.
     """
     # Get section and option from first argument.
-    arg1 = trigger.group(3).split('.')
+    if trigger.group(3) is not None:
+        arg1 = trigger.group(3).split('.')
+    else:
+        return bot.reply("Usage: .set section.option value")
```

Comment:
The `return` here is to stop the function, and not to return `bot.reply`'s result, and I'd rather see this instead:

```python
match = trigger.group(3)
if match is None:
    bot.reply("Usage: .set section.option value")
    return

args = match.split('.')
```